### PR TITLE
Add overdue list header to point out calls are free

### PR DESCRIPTION
**Pull request by rakshakhegde**
https://www.pivotaltracker.com/story/show/164695876

Icon color is also fixed.

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt [hunk 1]
```diff
@@ -28,138 +32,169 @@ class OverdueListAdapter @Inject constructor() : ListAdapter<OverdueListItem, Ov
...
-    }
-    removeFromListTextView.setOnClickListener {
-      eventStream.onNext(RemoveFromListClicked(appointment.appointmentUuid, appointment.patientUuid))
-    }
-  }
+sealed class OverdueListItem {
+
+  object OverdueHeaderItem : OverdueListItem()
+
+  data class OverduePatientItem(
```
Comment (saket) [inline app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt]: Considering that these are inner classes of `OverdueListItem`, prefixing them with `Overdue` is redundant. `OverdueListItem.Header` would be better than `OverdueListItem.OverdueHeaderItem`. Let's rename them?

File: app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt [hunk 2]
```diff
@@ -28,138 +32,169 @@ class OverdueListAdapter @Inject constructor() : ListAdapter<OverdueListItem, Ov
```
Comment (saket) [inline app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt]: There are some IDE inspection warnings in this class. Can you either suppress them or fix them?

File: app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt [hunk 3]
```diff
@@ -28,138 +32,169 @@ class OverdueListAdapter @Inject constructor() : ListAdapter<OverdueListItem, Ov
 
   val itemClicks = PublishSubject.create<UiEvent>()!!
 
+  companion object {
+    const val OVERDUE_HEADER = R.layout.item_overdue_list_header
+    const val OVERDUE_LIST = R.layout.item_overdue_list
```
Comment (saket) [inline app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt]: `OVERDUE_LIST` -> `OVERDUE_PATIENT`?

Comment (saket) [reply app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt]: Same feedback for the layout name

**Comment by rakshakhegde** [comment]
@saket Let me know once you have tested this on a device.

**Comment by saket** [comment]
@rakshakhegde oh yea that's already done. Sorry I forgot to tell you.

File: app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt (hunk 1 above)
Comment (rakshakhegde) [reply app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt]: Done

File: app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt (hunk 2 above)
Comment (rakshakhegde) [reply app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt]: Made functions `private`. Let me know if there are others.

File: app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt (hunk 3 above)
Comment (rakshakhegde) [reply app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt]: Done

**Pull request merged**